infrastructure/local/docker-compose.cnw.yml

```
@@ -29,14 +13,14 @@ services:
       NODE_ENV: development
       LOG_LEVEL: debug
       PORT: 80
-      SEED_FILE_LOCATION: /workspace/seed.primary.yml
+      SEED_FILE_LOCATION: /workspace/seed.yml
       KEY_FILE: /workspace/private-key.pem
     volumes:
-      - ./seed.primary.yml:/workspace/seed.primary.yml
-      - ./private-key.pem:/workspace/private-key.pem
+      - ./seed.cnw.yml:/workspace/seed.yml
+      - ./private-key.cnw.pem:/workspace/private-key.pem
     depends_on:
-      - backend
-  backend:
+      - cloud-nine-wallet-backend
+  cloud-nine-wallet-backend:
```

What if we add `cloud-nine-wallet` and `happy-life-bank` directories to `infrastructure/local`?
Then we can have standard `docker-compose.yml` file names and simplify the defined service names:
```suggestion
  backend:
```
since I think docker compose prepends service names with that of the directory

---

I tried that but when you specify 2 docker-compose files in the start command, they get merged and all the volumes are put into the directory that belongs to the first docker-compose file. So my keys and seeds were either overridden or not found, same with the seeds. 
Also, you have to do a structure like `cloud-nine-wallet/local/...` to keep the `local` prefix on the containers. Otherwise, the get the directory name the first docker-compose file is in as prefix. I had containers that were called `cloud-nine-wallet_happy-life-bank-auth`, etc.

---

could we change the localenv scripts to have separate `docker-compose` commands
```
    "cloud-nine": "docker compose -f ./infrastructure/local/cloud-nine/docker-compose.yml -f ./infrastructure/local/tigerbeetle_docker-compose.yml --env-file ./infrastructure/local/.env.tigerbeetle",
    "happy-bank": "docker compose -f ./infrastructure/local/happy-bank/docker-compose.yml",
    "localenv:start": "pnpm cloud-nine up -d && pnpm happy-bank up -d",
```
not sure if the tigerbeetle stuff would need relocating

---

If the networking still works, I guess we could do that. I'd assume the tigerbeetle stuff would just be its own thing, too and we'd have 
```
    "localenv:start": "pnpm cloud-nine up -d && pnpm happy-bank up -d" && tigerbeetle up -d,
```

---

Separate `up` commands would also probably fix 
- https://github.com/interledger/rafiki/issues/1003